app/views/layouts/angular-bootstrap/_endpoints_angular.html.haml

```diff
@@ -33,7 +34,7 @@
                        "selectpicker-for-select-tag" => "",
                        "prefix"                      => prefix.to_s,
                        "reset-validation-status"     => "#{prefix}_auth_status")
-    .col-md-8{"ng-if" => "emsCommonModel.emstype == 'nuage_network'"}
+    .col-md-8{"ng-if" => "emsCommonModel.emstype == 'nuage_network' || emsCommonModel.emstype == 'redfish_ph_infra'"}
```

Comment:
I would separate this now. Even if the set of security options is compatible for Nuage and Redfish, the latter does not support AMQP for the eventing (the next row checks whether the security protocols are displayed for AMQP tab or for the main tab; or at least, this is my understanding).

Thus, I suggest we create a new block specifically for the Redfish provider. Once events are going to be supported, we will add the security options for that as well (if necessary).

Reply:
This was a remnant from "I just want something to show on UI" times that I forgot to clean up properly. I will fix this by creating new Redfish-only block that will use dedicated security protocol list.